### PR TITLE
added a robust vision benchmark example submission

### PR DESCRIPTION
**Pull request by jonasrauber**
This pull request includes wrappers that make it easy to submit a CleverHans attack to the [`Robust Vision Benchmark`](https://robust.vision/benchmark) and an example submission.

### CODE REVIEW

File: examples/robust_vision_benchmark/cleverhans_attack_example/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+numpy
```
Comment (npapernot) [inline examples/robust_vision_benchmark/cleverhans_attack_example/requirements.txt]: Why are you including a requirements.txt file here?

Comment (wielandbrendel) [reply examples/robust_vision_benchmark/cleverhans_attack_example/requirements.txt]: The requirements.txt is only used in the Dockerfile and is part of the submission. We could make it part of the Dockerfile if you prefer to avoid confusion / reduce the number of files.

Comment (npapernot) [reply examples/robust_vision_benchmark/cleverhans_attack_example/requirements.txt]: It should be fine to leave like that, thanks for clarifying.

File: examples/robust_vision_benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## CleverHans Example Submission to the Robust Vision Benchmark
+
+Using the wrappers in this repository, submitting a CleverHans attack to the [Robust Vision Benchmark](https://robust.vision/benchmark) requires just a few lines of code:
+
+```python
```
Comment (npapernot) [inline examples/robust_vision_benchmark/README.md]: Could you remove the code snippet from the readme and just point to the file in the `cleverhans_attack_example` folder? We've had issues in the past with example code copied in markdown files getting out-dated.

File: examples/robust_vision_benchmark/cleverhans_attack_example/utils.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    tf.RegisterGradient(rnd_name)(grad)
+    g = tf.get_default_graph()
+    with g.gradient_override_map({
+            "PyFunc": rnd_name,
+            "PyFuncStateless": rnd_name}):
+        return tf.py_func(func, inp, Tout, stateful=stateful, name=name)
+
+
+class RVBCleverhansModel(cleverhans.model.Model):
```
Comment (npapernot) [inline examples/robust_vision_benchmark/cleverhans_attack_example/utils.py]: I see one issue with this wrapper is that some attacks will try to use the probabilities (post-softmax) instead of the logits. In its current form, the method `get_logits()` will work but not `get_probs()`. Could you also provide access to the probabilities?

Comment (wielandbrendel) [reply examples/robust_vision_benchmark/cleverhans_attack_example/utils.py]: We didn't implement get_probs because the Cleverhans model anyway falls back to softmax(logits) if probabilities are not defined, right?

Comment (npapernot) [reply examples/robust_vision_benchmark/cleverhans_attack_example/utils.py]: Not necessarily. If you take a look at the FastGradientMethod class, since you are passing it a `cleverhans.model.Model` object, it will call `model.get_probs()`: https://github.com/tensorflow/cleverhans/blob/master/cleverhans/attacks.py#L283

Comment (wielandbrendel) [reply examples/robust_vision_benchmark/cleverhans_attack_example/utils.py]: But that should be fine since RBVCleverhansModel is derived from cleverhans.model.Model, and the latter implements get_prob such that it computes softmax(logits) if no layer with the name 'probs' exists: https://github.com/tensorflow/cleverhans/blob/master/cleverhans/model.py#L58-L60 Or am I missing something?

Comment (npapernot) [reply examples/robust_vision_benchmark/cleverhans_attack_example/utils.py]: That's correct, my bad I forgot about that default implementation we added

File: examples/robust_vision_benchmark/README.md (hunk 2 above)
Comment (npapernot) [reply examples/robust_vision_benchmark/README.md]: Can you address this? We can merge afterwards

Comment (jonasrauber) [reply examples/robust_vision_benchmark/README.md]: Done
